services/jira/jira-sprint.service.js

```diff
@@ -106,6 +106,7 @@ module.exports = class JiraSprint extends BaseJsonService {
       if (issue.fields.resolution != null) {
         return issue.fields.resolution.name !== 'Unresolved'
       }
+      return undefined
```

Comment:
Think this captures the idea better:

```suggestion
      return false
```